web-server/views/users/profile.jade

```diff
@@ -0,0 +1,31 @@
+extends ../layout
+
+block content
+
+  .row
+    .col-md-4
+      h1= user.name
+      ul.list-inline
+        li
+          a(href=user.profileUrl target="_blank") @#{user.handle}
+        li
+          a(href=user.echoProfileURL target="_blank") Echo
+        li
+          a(href="https://curriculum.learnersguild.org/phases/4/") Phase: 3 (7/8)
+      img.img-thumbnail.user-avatar.user-avatar-huge(src="#{user.avatarUrl}" alt="avatar")
+    .col-md-8.profile-content
```

Comment:
Are you using `.profile-content` anywhere? does it have styles or js attached to it?

Reply:
ahh yes! I did take this out as well as the associated styles file.